[2025-03-14T01:04:42.103+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-03-14T01:04:42.154+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sales_pipeline.Gold_layer.Fact_sales scheduled__2025-03-14T00:45:00+00:00 [queued]>
[2025-03-14T01:04:42.173+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sales_pipeline.Gold_layer.Fact_sales scheduled__2025-03-14T00:45:00+00:00 [queued]>
[2025-03-14T01:04:42.174+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2025-03-14T01:04:42.202+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): Gold_layer.Fact_sales> on 2025-03-14 00:45:00+00:00
[2025-03-14T01:04:42.213+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2248) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-14T01:04:42.215+0000] {standard_task_runner.py:63} INFO - Started process 2255 to run task
[2025-03-14T01:04:42.214+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'sales_pipeline', 'Gold_layer.Fact_sales', 'scheduled__2025-03-14T00:45:00+00:00', '--job-id', '232', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmppcks5dkj']
[2025-03-14T01:04:42.218+0000] {standard_task_runner.py:91} INFO - Job 232: Subtask Gold_layer.Fact_sales
[2025-03-14T01:04:42.286+0000] {task_command.py:426} INFO - Running <TaskInstance: sales_pipeline.Gold_layer.Fact_sales scheduled__2025-03-14T00:45:00+00:00 [running]> on host 2b35ac7f54da
[2025-03-14T01:04:42.425+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Mahmoud' AIRFLOW_CTX_DAG_ID='sales_pipeline' AIRFLOW_CTX_TASK_ID='Gold_layer.Fact_sales' AIRFLOW_CTX_EXECUTION_DATE='2025-03-14T00:45:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-14T00:45:00+00:00'
[2025-03-14T01:04:42.426+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-03-14T01:04:42.443+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-14T01:04:42.444+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'dbt run --full-refresh --profiles-dir /opt/***/dbt --project-dir /opt/***/dbt/sales --models Fact_sales']
[2025-03-14T01:04:42.464+0000] {subprocess.py:86} INFO - Output:
[2025-03-14T01:04:46.434+0000] {subprocess.py:93} INFO - [0m01:04:46  Running with dbt=1.9.3
[2025-03-14T01:04:47.441+0000] {subprocess.py:93} INFO - /home/***/.local/lib/python3.12/site-packages/snowflake/connector/options.py:108: UserWarning: You have an incompatible version of 'pyarrow' installed (19.0.1), please install a version that adheres to: 'pyarrow<19.0.0; extra == "pandas"'
[2025-03-14T01:04:47.442+0000] {subprocess.py:93} INFO -   warn_incompatible_dep(
[2025-03-14T01:04:48.341+0000] {subprocess.py:93} INFO - [0m01:04:48  Registered adapter: snowflake=1.9.2
[2025-03-14T01:04:49.416+0000] {subprocess.py:93} INFO - [0m01:04:49  Found 10 models, 41 data tests, 6 sources, 855 macros
[2025-03-14T01:04:49.420+0000] {subprocess.py:93} INFO - [0m01:04:49
[2025-03-14T01:04:49.420+0000] {subprocess.py:93} INFO - [0m01:04:49  Concurrency: 2 threads (target='dev')
[2025-03-14T01:04:49.421+0000] {subprocess.py:93} INFO - [0m01:04:49
[2025-03-14T01:04:51.290+0000] {subprocess.py:93} INFO - [0m01:04:51  1 of 1 START sql view model layer_gold.Fact_sales .............................. [RUN]
[2025-03-14T01:04:51.837+0000] {subprocess.py:93} INFO - [0m01:04:51  1 of 1 OK created sql view model layer_gold.Fact_sales ......................... [[32mSUCCESS 1[0m in 0.54s]
[2025-03-14T01:04:52.156+0000] {subprocess.py:93} INFO - [0m01:04:52
[2025-03-14T01:04:52.156+0000] {subprocess.py:93} INFO - [0m01:04:52  Finished running 1 view model in 0 hours 0 minutes and 2.73 seconds (2.73s).
[2025-03-14T01:04:52.301+0000] {subprocess.py:93} INFO - [0m01:04:52
[2025-03-14T01:04:52.302+0000] {subprocess.py:93} INFO - [0m01:04:52  [32mCompleted successfully[0m
[2025-03-14T01:04:52.303+0000] {subprocess.py:93} INFO - [0m01:04:52
[2025-03-14T01:04:52.304+0000] {subprocess.py:93} INFO - [0m01:04:52  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-03-14T01:04:53.469+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-03-14T01:04:53.470+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-03-14T01:04:53.543+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=sales_pipeline, task_id=Gold_layer.Fact_sales, execution_date=20250314T004500, start_date=20250314T010442, end_date=20250314T010453
[2025-03-14T01:04:53.651+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-03-14T01:04:53.675+0000] {taskinstance.py:3482} INFO - 4 downstream tasks scheduled from follow-on schedule check
[2025-03-14T01:04:53.710+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
